Sources/PromiseKit/Cancellation/CancellableThenable.swift

```diff
@@ -296,48 +296,55 @@ public extension CancellableThenable {
 
 public extension CancellableThenable {
     /**
-     - Returns: The error with which this cancellable promise was rejected; `nil` if this promise is not rejected.
-     */
-    var error: Error? {
-        return thenable.error
-    }
-
-    /**
-     - Returns: `true` if the cancellable promise has not yet resolved.
+     - `true` if the cancellable promise has not yet resolved.
      */
     var isPending: Bool {
         return thenable.isPending
     }
 
     /**
-     - Returns: `true` if the cancellable promise has resolved.
+     - `true` if the cancellable promise has resolved.
```

Comment:
Why are we removing the documentation markers? This will cause our generated documentation to render incorrectly.

Reply:
Actually, in Xcode it shows incorrectly when using `Returns` keyword. This is a computed property and it does not use this keyword.
With `Returns:`
![with](https://user-images.githubusercontent.com/1700160/125692795-9dd73fe2-a63e-4f1c-b910-8eff4f930a06.png)
Without `Returns:`
![without](https://user-images.githubusercontent.com/1700160/125692814-7a5e6ade-48a8-4d76-96f9-413d0202ccd1.png)
Now I think we could even remove the dash "-":
![dash](https://user-images.githubusercontent.com/1700160/125692968-5834a910-fea5-47be-9290-2b832c51bce1.png)
What do you think? Do you use any other tool for generating documentation? I think Xcode quick-help documentation should be prioritized.

Reply:
Jazzy renders it:

https://mxcl.dev/PromiseKit/reference/v7/Protocols/CancellableThenable.html#/s:10PromiseKit19CancellableThenablePAAE5value1U_1TQZSgvp

https://github.com/mxcl/PromiseKit/blob/master/Sources/PromiseKit/Cancellation/CancellableThenable.swift#L333-L338

If we need to fix documentation it needs to be a separate PR.

While I appreciate this is more work for contribution, my main priority with this project is maintaining quality, not encouraging contribution. Your contributions are sincerely appreciated of course, but this project has tens of thousands of users, so my obligation is to them.

Reply:
But it will still work with jazzy, it's a win-win change.
Result is without dash, value is with a dash:
![jazzy](https://user-images.githubusercontent.com/1700160/125866090-7cb99fab-c777-4554-800e-401ad761b456.png)

Reply:
According to docs I could find `- Returns:` is correct, if Xcode doesn’t render it, it's an Xcode bug.

Please keep PRs focused on their task, tidying things up and fixing other things should be separate PRs to prevent the PR being 2 months of discussion on unrelated changes before we can actually review the code changes.

Case in point, this discussion here.

Reply:
As you wish.